features/steps/cocoa_steps.rb

```diff
@@ -44,45 +44,6 @@ def request_fields_are_equal(key, index_a, index_b)
   )
 end
 
-def check_device_model(field, list)
```

Comment:
BitBar name their devices differently, so there's no way to get a reliable handle on the model name using Appium.  I've therefore removed all of this functionality and now just match the fields against a regex.  If the regex matches it seems very unlikely that the notifier will have generated an incorrect value.